migrations/database_migration.py

```diff
@@ -0,0 +1,136 @@
+import json
+import sqlite3
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import sessionmaker
+from database.database_tables import Base, User, Role, Balance, Operation, UserWeekStat
+from config.database_config import DATABASE_FILE_PATH, DB_HOST, DB_PORT, DB_USER, DB_PASS, DB_NAME
```

Comment:
Database does not have a password, but the database user does. Please rename this parameter